extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts

```diff
@@ -119,6 +119,7 @@ export class ConfigurePythonWizard {
 			return true;
 		});
 
+		this._wizard.nextButton.enabled = false;
```

Comment:
For this particular scenario, any thoughts on if we should log when the next button is enabled, and when the rest of the page has loaded (just for sanity)?